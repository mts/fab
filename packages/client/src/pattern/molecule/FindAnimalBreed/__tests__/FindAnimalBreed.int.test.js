import { findDogBreedRender, findCatBreedRender } from './FindAnimalBreed.int.render'

jest.mock('../../../../store/thunk/setRandomImage', () => ({ setDogRandomImageThunk: () => {}, setCatRandomImageThunk: () => {} }))

describe('<FindAnimalBreed />', () => {
  describe('Snaphot', () => {
    test('must match findDogBreedRender', () => {
      expect(global.renderToJSON(findDogBreedRender)).toMatchSnapshot()
    })

    test('must match findCatBreedRender', () => {
      expect(global.renderToJSON(findCatBreedRender)).toMatchSnapshot()
    })
  })
})
