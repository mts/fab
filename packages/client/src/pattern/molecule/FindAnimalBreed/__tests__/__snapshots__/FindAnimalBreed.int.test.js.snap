// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FindAnimalBreed /> Snaphot must match findCatBreedRender 1`] = `
<div
  className="breed"
>
  <h2>
    A random image of 
     breed
  </h2>
  <h4>
    Type in new breed, press enter or click search
  </h4>
  <div
    className="breedSearch"
  >
    <input
      className="input"
      onKeyUp={[Function]}
    />
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Search Cat
    </button>
  </div>
  <div
    className="breedShow"
  >
    <div
      className="image"
    >
      <img
        alt="beng"
        src="https://66.media.tumblr.com/tumblr_m4iejdP8Hb1qd477zo1_250.jpg"
      />
    </div>
  </div>
</div>
`;

exports[`<FindAnimalBreed /> Snaphot must match findDogBreedRender 1`] = `
<div
  className="breed"
>
  <h2>
    A random image of 
     breed
  </h2>
  <h4>
    Type in new breed, press enter or click search
  </h4>
  <div
    className="breedSearch"
  >
    <input
      className="input"
      onKeyUp={[Function]}
    />
    <button
      className="button"
      onClick={[Function]}
      type="button"
    >
      Search Dog
    </button>
  </div>
  <div
    className="breedShow"
  >
    <div
      className="image"
    >
      <img
        alt="collie"
        src="https://images.dog.ceo/breeds/collie-border/n02106166_1031.jpg"
      />
    </div>
  </div>
</div>
`;
